Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import src.modeling
  File "/users/acr23hk/paper/fsod-dc/src/modeling/__init__.py", line 1, in <module>
    from . import rcnn, roi_heads
  File "/users/acr23hk/paper/fsod-dc/src/modeling/rcnn.py", line 10, in <module>
    from .refine import Refine
  File "/users/acr23hk/paper/fsod-dc/src/modeling/refine.py", line 81
    if self.momentum == 
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/users/acr23hk/.conda/envs/paper1/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/users/acr23hk/.conda/envs/paper1/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/users/acr23hk/paper/fsod-dc/tools/ckpt_surgery.py", line 128, in <module>
    main(args)
  File "/users/acr23hk/paper/fsod-dc/tools/ckpt_surgery.py", line 83, in main
    sd = torch.load(inp_path, map_location="cpu")
  File "/users/acr23hk/.conda/envs/paper1/lib/python3.8/site-packages/torch/serialization.py", line 594, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/users/acr23hk/.conda/envs/paper1/lib/python3.8/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/users/acr23hk/.conda/envs/paper1/lib/python3.8/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/users/acr23hk/paper/fsod-dc/checkpoints/coco/1726447908/base/model_final.pth'
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import src.modeling
  File "/users/acr23hk/paper/fsod-dc/src/modeling/__init__.py", line 1, in <module>
    from . import rcnn, roi_heads
  File "/users/acr23hk/paper/fsod-dc/src/modeling/rcnn.py", line 10, in <module>
    from .refine import Refine
  File "/users/acr23hk/paper/fsod-dc/src/modeling/refine.py", line 81
    if self.momentum == 
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import src.modeling
  File "/users/acr23hk/paper/fsod-dc/src/modeling/__init__.py", line 1, in <module>
    from . import rcnn, roi_heads
  File "/users/acr23hk/paper/fsod-dc/src/modeling/rcnn.py", line 10, in <module>
    from .refine import Refine
  File "/users/acr23hk/paper/fsod-dc/src/modeling/refine.py", line 81
    if self.momentum == 
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import src.modeling
  File "/users/acr23hk/paper/fsod-dc/src/modeling/__init__.py", line 1, in <module>
    from . import rcnn, roi_heads
  File "/users/acr23hk/paper/fsod-dc/src/modeling/rcnn.py", line 10, in <module>
    from .refine import Refine
  File "/users/acr23hk/paper/fsod-dc/src/modeling/refine.py", line 81
    if self.momentum == 
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import src.modeling
  File "/users/acr23hk/paper/fsod-dc/src/modeling/__init__.py", line 1, in <module>
    from . import rcnn, roi_heads
  File "/users/acr23hk/paper/fsod-dc/src/modeling/rcnn.py", line 10, in <module>
    from .refine import Refine
  File "/users/acr23hk/paper/fsod-dc/src/modeling/refine.py", line 81
    if self.momentum == 
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import src.modeling
  File "/users/acr23hk/paper/fsod-dc/src/modeling/__init__.py", line 1, in <module>
    from . import rcnn, roi_heads
  File "/users/acr23hk/paper/fsod-dc/src/modeling/rcnn.py", line 10, in <module>
    from .refine import Refine
  File "/users/acr23hk/paper/fsod-dc/src/modeling/refine.py", line 81
    if self.momentum == 
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    import src.modeling
  File "/users/acr23hk/paper/fsod-dc/src/modeling/__init__.py", line 1, in <module>
    from . import rcnn, roi_heads
  File "/users/acr23hk/paper/fsod-dc/src/modeling/rcnn.py", line 10, in <module>
    from .refine import Refine
  File "/users/acr23hk/paper/fsod-dc/src/modeling/refine.py", line 81
    if self.momentum == 
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/users/acr23hk/.conda/envs/paper1/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/users/acr23hk/.conda/envs/paper1/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/users/acr23hk/paper/fsod-dc/tools/display_results.py", line 189, in <module>
    main(args)
  File "/users/acr23hk/paper/fsod-dc/tools/display_results.py", line 128, in main
    table = open(os.path.join(root, "results.txt"), "w")
FileNotFoundError: [Errno 2] No such file or directory: 'checkpoints/coco/1726447908/results.txt'
